examples/src/test/scala/docs/cassandra/CassandraProjectionDocExample.scala

```diff
@@ -71,9 +84,58 @@ class CassandraProjectionDocExample {
     //#atMostOnce
     val projection =
       CassandraProjection.atMostOnce(
-        projectionId = ProjectionId("ShoppingCarts", "carts-1"),
+        projectionId = ProjectionId("shopping-carts", "carts-1"),
         sourceProvider,
         handler = new ShoppingCartHandler)
     //#atMostOnce
   }
+
+  object IllustrateRunningWithShardedDaemon {
+
+    //#running-source-provider
+    def sourceProvider(tag: String) =
+      EventSourcedProvider
+        .eventsByTag[ShoppingCart.Event](
+          systemProvider = system,
+          readJournalPluginId = CassandraReadJournal.Identifier,
+          tag = tag)
+    //#running-source-provider
+
+    //#running-projection
+    def projection(tag: String) =
+      CassandraProjection.atLeastOnce(
+        projectionId = ProjectionId("shopping-carts", tag),
+        sourceProvider(tag),
+        saveOffsetAfterEnvelopes = 100,
+        saveOffsetAfterDuration = 500.millis,
+        handler = new ShoppingCartHandler)
+    //#running-projection
+
+    //#running-with-daemon-process
+    ShardedDaemonProcess(system).init[ProjectionBehavior.Command](
+      name = "shopping-carts",
+      numberOfInstances = ShoppingCart.tags.size,
+      behaviorFactory = n => ProjectionBehavior(projection(ShoppingCart.tags(n))),
+      settings = ShardedDaemonProcessSettings(system),
+      stopMessage = Some(ProjectionBehavior.Stop))
+    //#running-with-daemon-process
+  }
```

Comment:
When I look to this, it looks very clustered. The are lots of concepts that a user needs to know and write together. 

* types - the user needs to know the types they must pick from `ProjectionBehavior` and wired it correctly. 
* they need to pass the settings, even if they don't tweak it, because they need to pass the stop message
* user may choose to omit the settings and the stop message, while in the case of Projection we pretty much want them to use the Stop message.

I do think there is room for a `ProjectionDaemon`

```scala
def projection(tag: String) =
      CassandraProjection.atLeastOnce(
        projectionId = ProjectionId("shopping-carts", tag),
        sourceProvider(tag),
        saveOffsetAfterEnvelopes = 100,
        saveOffsetAfterDuration = 500.millis,
        handler = new ShoppingCartHandler)

val tags = List("tag-1", "tag-2", "tag-3")
val projections = tags.map(projection)

ProjectionDaemon(system).init(
  name = "shopping-carts",
  projections = projections
)
```

We don't need the types, we don't need `numberOfInstances` nor `behaviorFactory`. We can do it now because projection is a factory of running projections.